docusaurus/docs/publish-a-plugin/publish-or-update-a-plugin.md

```diff
@@ -111,9 +117,9 @@ To submit an **update** for an already published plugin:
 - The plugin's catalog page content is extracted from the plugin README file.
   To update the plugin's catalog page, submit an updated plugin with the new content included in the README file.
 
-### Can I unlist my plugin from the Grafana plugin catalog in case of a bug?
+### Can I unlist a plugin? How are plugins deprecated?
 
-- In the event of a bug, unlisting the plugin from our catalog may be possible in exceptional cases, such as security concerns. However, we don't have control over the instances where the plugin is installed.
+- Refer to the Grafana Labs [Plugin Deprecation Policy](https://grafana.com/legal/plugin-deprecation/) to learn more about plugin deprecation.
 
 ### Can I distribute my plugin somewhere else other than the Grafana plugin catalog?
```

Comment:
I know this haven't changed, but wonder if we want to push some more for using our catalog and less on the alternative methods.

Reply:
I see what you mean. I've separated the "unlist" question from "deprecated" and moved the latter to later in the FAQ.